src/system/PermissionsModule/Tests/Api/Fixtures/StubPermissionRepository.php

```diff
@@ -100,6 +95,12 @@ public function getAllComponents(): array
         return [];
     }
 
+    public function getAllColours(): array
+    {
+        // TODO: Implement getAllColours() method.
```

Comment:
?
either implement or remove comment

Reply:
the whole stub is full of these comments, I just copied `getAllComponents`

Reply:
ok. will you just remove all the comments then? 😄

Reply:
yes; done